articles/quickstart.md

```diff
@@ -71,7 +71,7 @@ The `Bell.qs` file should have the following contents:
 ```qsharp
 namespace Quantum.Bell
 {
-    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Standard;
```

Comment:
No functions or operations outside of the intrinsics provided with the prelude are used with in this snippet, maybe it makes sense to drop the `open` statement altogether here?
```suggestion
```